firefox_ui_tests/functional/locationbar/test_suggest_bookmarks.py

```diff
@@ -72,15 +72,16 @@ def visit_urls():
 
         # Wait for the search string to be present, for the autocomplete results to appear
         # and for there to be exactly one autocomplete result
-        self.wait_for_condition(lambda mn: locationbar.value == search_string)
-        self.wait_for_condition(lambda mn: autocomplete_results.is_open)
-        self.wait_for_condition(lambda mn: len(autocomplete_results.visible_results) == 1)
+        Wait(self.marionette).until(lambda mn: locationbar.value == search_string)
+        Wait(self.marionette).until(lambda mn: autocomplete_results.is_open)
+        Wait(self.marionette).until(
+            lambda mn: len(autocomplete_results.visible_results) == 1)
```

Comment:
Here I would also combine the autocomplete conditions into a single `until` call.